v3/CHANGELOG.md

```diff
@@ -1,4 +1,24 @@
 # Changelog
+## Version 3.0.0-pre.1518 - March 14, 2024
+
+### Features/Improvements
+- PT-187203297: Dragging a color attribute to graph misbehaves
+- PT-187142716: Bar for Each Point configuration option breaks axis
```

Comment:
I would put these two in the bug fixes category.

Reply:
@kswenson - great! I'll make those changes and then do the merge.